supplier_app/views.py

```diff
@@ -163,6 +164,13 @@ def form_valid(self, forms):
             company = Company.objects.get(companyuserpermission__user=self.request.user)
             taxpayer.company = company
             taxpayer.save()
+            eb_entities = forms['taxpayer_form'].cleaned_data['eb_entities']
+            for eb_entity in eb_entities:
+                eb_entity = EBEntity.objects.get(pk=eb_entity.id)
+                TaxPayerXEBEntity.objects.create(
```

Comment:
Rename `TaxPayerXEBEntity` to `TaxPayerEBEntity`

Reply:
Ok i will change that